R/spq_add.R

```diff
@@ -1,23 +1,24 @@
-#' Add a triplet statement to a query
+#' Add a triple statement to a query
 #' @param query query
-#' @param triplet the triplet statement (replaces arguments subject verb and object)
-#' @param subject an anonymous variable (for instance, and by default, "?subject") or item (for instance "wd:Q456"))
-#' @param verb the property (for instance "wdt:P190")
-#' @param object an anonymous variable (for instance, and by default, "?object") or item (for instance "wd:Q456"))
-#' @param required whether the existence of a value for the triplet is required or not (defaults to TRUE).
-#'   If set to FALSE, then other triplets in the query are returned even if this particular triplet is missing)
-#' @param label a vector of variables for which to include a label column (defaults to NA)
-#' @param within_box if provided, rectangular bounding box for the triplet query.
+#' @param ... the triple statement (replaces arguments subject verb and object)
+#' Either R-DSL (or SPARQL strings escaped with `spq()`, or strings, see examples)
+#' @param .subject an anonymous variable (for instance, and by default, "?subject") or item (for instance "wd:Q456"))
```

Comment:
with dots as in dplyr, for avoiding clash in names with what the user provides in `...`